### PR TITLE
Updating OCP3 internal images for OCP4 cluster (1.4.1)

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-529

MTC 1.4.1 (Mar 3, but can be merged earlier)

CP to 4.5, 4.6, 4.7

### CODE REVIEW

**Comment by XinRedhat** [comment]
LGTM

File: modules/migration-updating-internal-images.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+.Procedure
+
+. Expose the internal registries on the source and target clusters.
+. If you are using insecure registries, add your registry host values to the `[registries.insecure]` section of the `/etc/container/registries.conf` to ensure that Podman does not encounter a TLS verification error.
+. Log in to the {product-title} 3 registry:
++
+[source,terminal]
+----
+$ podman login -u $(oc whoami) -p $(oc whoami -t) --tls-verify=false <ocp3-host>
```
Comment (bergerhoffer) [inline modules/migration-updating-internal-images.adoc:25]: Replaceable values should use underscores, so all instances of "<ocp3-host>" in this PR should be "<ocp3_host>". Same for ocp4_host

File: modules/migration-updating-internal-images.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+----
+
+. Tag the image for the {product-title} 4 registry:
++
+[source,terminal]
+----
+$ podman tag <ocp3-host>/openshift/python:3.3 <ocp4-host>/openshift/python:3.3
+----
+
+. Push the image to the OpenShift 4 registry:
```
Comment (bergerhoffer) [inline modules/migration-updating-internal-images.adoc:49]: Should "OpenShift" be {product-title} here too?

**Review by bergerhoffer — COMMENTED** [review]
Just a few things!

**Pull request merged**